relay/engines/docker.go

```diff
@@ -229,7 +229,31 @@ func (de *DockerEngine) attemptAuth() error {
 	return nil
 }
 
+func (de *DockerEngine) developerModeRefresh(bundle *config.Bundle) error {
+	if de.relayConfig.DevMode == true {
+		fullName := fmt.Sprintf("%s:%s", bundle.Docker.Image, bundle.Docker.Tag)
+		log.Warnf("Developer mode: Refreshing Docker image %s.", fullName)
+		err := de.pullImage(fullName)
+		if err != nil {
+			log.Errorf("Developer mode: Refresh of Docker image %s failed: %s.", fullName, err)
```

Comment:
I thought about doing that but wondered if it'd be overkill given warn and error are already color coded in the console.